samples/boards/nrf52/nffs/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(NONE)
+
+target_sources(app PRIVATE
+  src/main.c
+)
+
+
+zephyr_library_include_directories(
+	$ENV{ZEPHYR_BASE}/samples/bluetooth
```

Comment:
I don't understand why you need to include any bluetooth code here.

Reply:
I've removed it.